package.json

```diff
@@ -114,7 +114,7 @@
     }
   },
   "scripts": {
-    "version": "pnpm changeset version && pnpm install",
+    "version": "pnpm changeset version && pnpm install --no-frozen-lockfile",
```

Comment:
I'm not quite sure why `pnpm install` is even necessary for a `version` script?

Reply:
We need to commit lock file changes after version changes. Originally used —lock file-only but that had issues

Reply:
Ah, that makes sense!